src/Chapter2.hs

```diff
@@ -336,7 +358,10 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList start end list = 
+    if (start * end) < 0
```

Comment:
That's a clever check :smile: 
But you should be careful with arithmetic operations when working with bounded types like `Int` due to integer overflow:

```
ghci> 2 * 4611686018427387904 :: Int
-9223372036854775808
```

Reply:
Oh! I didn't think of that! Will keep this in mind next time I come up with clever hacks. :smile: